### ISSUE TITLE
i have a white tool bar in file explorer.

### ISSUE THREAD

**Issue by Remikaa** [Open-Shell/Open-Shell-Menu#1816]
### Describe the bug

i downloaded classic shell a while ago. After uninstalling, i had a tool bar issue. every time my wallpaper changes (since i have wallpaper engine) the toolbar returns even after locking it.
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/150489784/e777f594-5b36-43c4-9b83-f1aa0ab40f8d)
i tried locking the tool bar from the reg editor but it returns after the wallpaper change. i redownloaded open shell without the toolbar extension and then deleted but nothing happened.

### Area of issue

Taskbar, Windows Explorer

### To reproduce

1. Install classic shell
2. uninstall classic shell
3. lock the toolbar
4. open a file explorer tab (it wont have the tool bar) then minimize it.
5. let the wallpapers change
6. open the previously minimized tab and see the toolbar

### Open-Shell version

4.4.191

### Windows version

Windows 11 23H2

**Comment by ge0rdi** [comment]
I think this https://github.com/Open-Shell/Open-Shell-Menu/issues/326#issuecomment-813086363 should help you.

**Comment by ge0rdi** [comment]
> how do i add the reg key?

Maybe look up some guides about how to edit Windows registry?

> and i also have the toolbar locked anyways from the reg editor

What do you mean by that?

> everytime my wallpaper changes the toolbar is back

You mean that it works fine, but then that menu bar appears again?
I'm afraid this has nothing to do with `Open-Shell`. So not sure how to help :(

**Comment by Remikaa** [comment]
@ge0rdi so i have wallpaper engine. the bar isnt there when i initially open file explorer but when wallpaper engine changes my wallpaper and i open the previously opened file explorer tab i get the bar. the problem only started to happen after i got classic shell so im pretty sure it is its fault.

**Comment by lisansas** [comment]
Same here.

But I haven't changed the wallpaper.

Just doing some work, like unzipping some files, or playing a game for a while and it comes up

**Comment by LavaCreeperYT** [comment]
I've had the same issue on my previous install of 11, fixed it by disabling Open shell toolbar in Internet Properties

**Comment by LavaCreeperYT** [comment]
No, go to Programs tab, then press this button (idk how it's called in english windows, hope you can find it)
![image](https://github.com/user-attachments/assets/f83be477-cf30-483e-9ca7-c7a2e199a142)

**Comment by lisansas** [comment]
> 不，转到“程序”选项卡，然后按此按钮（idk在英文窗口中如何称呼，希望您能找到它） ![图像](https://private-user-images.githubusercontent.com/157400272/348696125-f83be477-cf30-483e-9ca7-c7a2e199a142.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzNzczOTEsIm5iZiI6MTcyMTM3NzA5MSwicGF0aCI6Ii8xNTc0MDAyNzIvMzQ4Njk2MTI1LWY4M2JlNDc3LWNmMzAtNDgzZS05Y2E3LWM3YTJlMTk5YTE0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxOVQwODE4MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNWYyYzU4NTVjYzZlYjgzNzRlOTRkZmM1NjkxNjZiNTc3YmEzMTU5ZTFiNTg1MmZkN2FhOGQ5ZmM3M2NiYTVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W3mUKKn2IOAzZLZ8GO0wyC5iilpAQHgyuOfNcJcr6m8)

it works!

love you bro.

**Comment by VMCpp** [comment]
there's no openshell add-on in there at all.

**Comment by LavaCreeperYT** [comment]
> there's no openshell add-on in there at all.
It should be named like 7Toolbar or something like this

**Comment by zachpoblete** [comment]
@VMCpp Did you reinstall Open Shell with Classical Explorer? That's how I saw them. @LavaCreeperYT they're already disabled (and there's no 7Toolbar).

![image](https://github.com/user-attachments/assets/6479d660-1a3e-4e8e-b7a8-de27f481a22b)

I guess I'll just see if reinstalling Open Shell while having the add-ons disabled will fix it, which would be weird solution.

**Comment by LavaCreeperYT** [comment]
> @VMCpp Did you reinstall Open Shell with Classical Explorer? That's how I saw them. @LavaCreeperYT they're already disabled (and there's no 7Toolbar).
> 
> ![image](https://private-user-images.githubusercontent.com/92368853/352065213-6479d660-1a3e-4e8e-b7a8-de27f481a22b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5NjQ3MjcsIm5iZiI6MTcyMTk2NDQyNywicGF0aCI6Ii85MjM2ODg1My8zNTIwNjUyMTMtNjQ3OWQ2NjAtMWEzZS00ZThlLWI3YTgtZGUyN2Y0ODFhMjJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDAzMjcwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxYjAzZTlhYjAwMzViNzUzZjM1Y2FhMTZiNDQ3N2RhMWQ5NTkxZjdkMjI0M2FmNWY0NDY5NWQ4ZTMzMjQ1YTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CKucPpg1ZOSgwaZfqiKheYGI7v-xHVNjGYd79IULEC0)
> 
> I guess I'll just see if reinstalling Open Shell while having the add-ons disabled will fix it, which would be weird solution.
> 
> I've had this white toolbar for more than a year, but it's only there _sometimes_. I've tried so many solutions, but none of them have worked so far. I did start encountering it some time after uninstalling Open Shell, but I'm really not sure if Open Shell was the cause.
oh yeah, disable Classic Explorer Bar. I just forgot it's name

**Comment by zachpoblete** [comment]
Yup, it just appeared again. Dang.

**Comment by ZeroFirewall** [comment]
I have tried everything and what solved my problem is Unchecking/Checking this option:
![2024-10-07 21_01_46-Folder Options](https://github.com/user-attachments/assets/e5f8871e-553b-419d-8885-b6d2177c1803)
It was unchecked by default but after applying changes and then unchecking it back, the white toolbar never showed up again.

**Comment by GregariousJB** [comment]
I also have this white bar issue after installing then re-installing without the Classic Explorer Bar selected in an attempt to remove it. The above suggestions have not fixed this.

![shit](https://i.imgur.com/FnsxKjj.png)

**Comment by Merigold117** [comment]
So has there still not a solution found to this. It's a really big eyesore to me whenever I open the file explorer.

**Comment by eucvm-lab** [comment]
Installed and uninstalled this program today and I have a white line under the File Explorer toolbar too. I did not configure the app at all - just uninstalled it immediately after the installation. The DWORD Locked=1 registry update worked immediately. It took me an hour of my life to realize this POS app was to blame.

**Comment by zachpoblete** [comment]
> The DWORD Locked=1 registry update worked immediately.

Where can I learn how to do this fix?
